scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
```

Comment:
TARS:<br/>:stop_sign: **Security Vulnerability**: Hardcoded password detected. This is a severe security risk[^1]. Storing passwords in plaintext within source code is extremely dangerous.

Recommendation: Use a secure password management system or environment variables to handle sensitive credentials.
```suggestion
// Remove this line and use a secure method to retrieve the password
// const password = process.env.ADMIN_PASSWORD // Example of using an environment variable
```

[^1]: CWE-798: Use of Hard-coded Credentials - https://cwe.mitre.org/data/definitions/798.html